public/components/content-list/content-list.js

```diff
@@ -107,6 +107,23 @@ function wfContentListController($rootScope, $scope, $anchorScroll, statuses, le
         wfFiltersService.clearAll(false);
     };
 
+    $scope.getSortDirection = columnName => columnName === $scope.sortColumn ? $scope.sortDirection : undefined;
+
+    $scope.sortColumn = undefined;
+    $scope.sortDirection = undefined;
+
+    const sortStates = ['asc', 'desc', undefined]
+    $scope.toggleSortState = columnName => {
+      // Reset the sort order if we're toggling a new field
```

Comment:
Reading this back, it's a bit golf-y – lmk if we'd like to express this in a simpler (but more verbose) way.